elastic/kibana
# [Security Solution] Enable entity previews in document details flyout

## Summary

This PR enables entity flyout in the alert flyout where host/user name is displayed. The entity flyout is rendered as a preview on top of the summary (right) panel. At the bottom of preview, there is a footer that let users open entity flyout as usual, which allows for expand/collapse functionality.

How to test:
- Enable feature flag `entityAlertPreviewEnabled`
- Generate some alerts and open alert flyout
- The following places should open host/user flyout as a preview
   - Summary panel (right)-> Investigation -> host and user names
   - Summary panel (right) -> Insights -> host and user names in entities
   - Details panel (left) -> Insights -> Entities -> the host and user names in the header, plus the related users and hosts in the table below
   - Details panel (left) -> Insights -> Prevalence -> host and user name


https://github.com/elastic/kibana/assets/18648970/68432768-f494-4f74-8ff8-d4dc6ca0c197



### Checklist
